docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
+
+## Prerequisites
+
+- Create a new [Etsy account](https://www.etsy.com/join) and store if you don’t have one.
+- Create a private app in your store by navigating to Apps, then Manage private apps.
+- Create a new private app, with any “Private app name” and leaving the default permissions as read access under Admin API.
+- Enable the Etsy API by checking the box that says “Allow this app to access your storefront data using Storefront API”.
+- Make sure to also grant access to read product and customer tags by checking their corresponding boxes.
+
+## If you do not already have one ready, [create a Gatsby site](/docs/quick-start):
```

Comment:
Creating a Gatsby site probably doesn't need to be linked _and_ explained in quite so much detail. It also probably doesn't need to be an `h2`. If you mention it at all, I'd shrink this whole section down to a link.